packages/fxa-settings/src/components/GetDataTrio/en-US.ftl

```diff
@@ -1,9 +1,8 @@
-# GetDataTrio component, part of Recovery Key flow
+# GetDataTrio component, part of Account Recovery Key flow
 
-get-data-trio-title = Recovery Codes
 get-data-trio-title-firefox = { -brand-firefox }
-get-data-trio-title-firefox-recovery-key = { -brand-firefox } recovery key
-get-data-trio-title-firefox-backup-verification-codes = { -brand-firefox } backup verification codes
+get-data-trio-title-firefox-recovery-key = { -brand-firefox } account recovery key
+get-data-trio-title-firefox-backup-verification-codes = { -brand-firefox } backup authentication codes
```

Comment:
Commenting for myself: these strings haven't been synced with Pontoon so no id change required.